integration-tests/data/mock_stack_analysis_v2_junit.json

```diff
@@ -127,7 +127,7 @@
                 "manifest_file_path": "/home/ptisnovs/bayesian",
                 "usage_outliers": [
                     {
-                        "outlier_prbability": 0.9912856374394844,
+                        "frequency_count": 0.9912856374394844,
```

Comment:
@tisnik can we update the values as well?

Reply:
to what? if I understand correctly, every value >= 0 is ok? Or must it be integer?

Reply:
It has to be an integer.